### PR TITLE
feat: DEV-2304: Implement health check for rqworker

### PR DESCRIPTION
**Pull request by KonstantinKorotaev**

### CODE REVIEW

File: label_studio/core/redis.py [hunk 1]
```diff
@@ -100,3 +102,30 @@ def start_job_async_or_sync(job, *args, **kwargs):
...
+    # check workers
+    workers = Worker.all(connection=_redis)
+    workers_count = len(workers)
+    if workers_count == 0:
+        return False, "No workers found. Start several workers."
+    if workers_count * RATIO < total_jobs:
+        error_text = f"Workers seems to be overloaded. {total_jobs} jobs in the queues."
+    # check if workers in correct state
+    if all([worker for worker in workers if worker.state in ['started', 'busy', 'idle']]):
+        return False, "No workers found. Start several workers."
```
Comment (makseq) [inline label_studio/core/redis.py:130]: change it to another text, it's the same as on the line 125, it's bad for debugging.

File: label_studio/core/views.py [hunk 2]
```diff
@@ -78,10 +87,16 @@ def version_page(request):
...
+    redis = bool_from_request(request.GET, "redis", False)
+    if not redis:
+        logger.debug('Got /health request.')
+        return HttpResponse(json.dumps({
+            "status": "UP"
+        }))
+    else:
+        health_state = rqworker_healthcheck()
+        result = {"message": health_state[1]}
+        return JsonResponse(result, status=200 if health_state[0] else 500)
```
Comment (makseq) [inline label_studio/core/views.py:99]: we need to return "status": "UP" as it was before, otherwise we will break a lot of our client's setups.

**Review by makseq — CHANGES_REQUESTED** [review]

File: label_studio/core/redis.py (hunk 1 above)
Comment (KonstantinKorotaev) [reply label_studio/core/redis.py]: Fixed

File: label_studio/core/views.py (hunk 2 above)
Comment (KonstantinKorotaev) [reply label_studio/core/views.py:99]: Fixed

**Review by makseq — APPROVED** [review]

**Review by farioas — CHANGES_REQUESTED** [review]
This will not work in the case of SSL enabled on redis or pgsql side

**Pull request closed without merging**